rustler_codegen/src/lib.rs

```diff
@@ -70,9 +70,9 @@ pub fn nif(args: TokenStream, input: TokenStream) -> TokenStream {
     nif::transcoder_decorator(args, input).into()
 }
 
-/// Implementation of the `NifStruct` macro that lets the user annotate a struct that will
-/// be translated directly from an Elixir struct to a Rust struct. For example, the following
-/// struct, annotated as such:
+/// Create a correspondence between an Elixir struct and a Rust struct.
```

Comment:
> Macro for automatically deriving `Encoder` and `Decoder` implementations which convert between a Rust struct and an Elixir struct. 

How about something like this? In my mind the word correspondence might not be the most descriptive in this case.

Reply:
The same also applies in the other cases this wording is used.

Reply:
That's definitely an improvement! I changed the wording further, hopefully keeping your spirit.